src/util.js

```diff
@@ -101,39 +102,69 @@ define([
         };
         /**
          * Register an event handler to be called each time an event is fired.
+         *
+         * @param type - Event type string.
+         * @param method - Event handler method.
+         * @param parameters - Parameters to be passed to method. If null
```

Comment:
no action required, but there may be a time when you need both parameters and the event object